Examples/create-app.js

```diff
@@ -35,6 +35,15 @@ if (fs.existsSync(appName)) {
 let appNameAndroid = `${appName}-android`;
 let appNameIOS = `${appName}-ios`;
 let reactNativeVersion = args[1] || `react-native@${execSync('npm view react-native version')}`.trim();
+
+let reactNativeVersionBellowV049;
+if (reactNativeVersion) {
+    let reactNativeVersionNumberString = reactNativeVersion.split("@")[1];
+    reactNativeVersionBellowV049 = reactNativeVersionNumberString.split('.')[1] < 49
+} else {
+    reactNativeVersionBellowV049 = false;
```

Comment:
I'm no sure if we need this condition at all. 
1) We can assign `reactNativeVersionBellowV049` false by default
2) If we do not have `reactNativeVersion` specified may be it make sense to terminate the script and provide an error? Will the code  below work if we do not specify rn version?

Reply:
1. Yes, I can do this, it will be ```false``` by default.
1. If ```reactNativeVersion``` isn't specified, then script will just use latest version:
```
reactNativeVersion = args[1] || `react-native@${execSync('npm view react-native version')}`.trim();
```